src/library/scala/util/control/Exception.scala

```diff
@@ -22,7 +22,7 @@ import scala.language.implicitConversions
  *  Each class is independently composable.
  *
  *  This class differs from [[scala.util.Try]] in that it focuses on composing exception handlers rather than
- *  composing behavior.  All behavior should be composed first and fed to a [[Catch]] object using one of the
+ *  composing behavior.  All behavior should be composed first and fed to a [[scala.util.control.Exception.Catch]] object using one of the
```

Comment:
Might still want the shorter display text

```suggestion
 *  composing behavior.  All behavior should be composed first and fed to a [[scala.util.control.Exception.Catch Catch]] object using one of the
```

Reply:
Okay, but the shorter one seems an error.